php/swoole/composer.json

```
@@ -0,0 +1,4 @@
+{
+  "require": {
+  }
+}
```

I suggest having the following manifest
```suggestion
{
  "require": {
    "ext-swoole": "~4.6.0",
  }
}
```
it ensure that we are in-sync with version in the doc